clearblade/cloud/iot_v1/device_types.py

```diff
@@ -1,7 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 ClearBlade Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 from typing import List
 from .resources import GatewayType, LogLevel
 from .utils import get_value
-
+import os
+from google.api_core.datetime_helpers import DatetimeWithNanoseconds
```

Comment:
I think this looks external library because when I tried to use that I got this alert,
Import "google.api_core.datetime_helpers" could not be resolvedPylance[reportMissingImports](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportMissingImports)

Reply:
Yes, I see. Would it make sense to add a note in the README to say IF you want that type for times, THEN you should run **pip install google-api-core**? See here: https://pypi.org/project/google-api-core/

I think I happened to have this already installed because I have used the Google IoTCore Python SDK before.

Reply:
We can update the dependencies inside setup.py to add this library. That should download the package when installing the sdk.

Reply:
I went ahead and added the note to the README. Does that help @ronak-ingress ?

Reply:
I just saw @rajasd27 note. I'll make changes accordingly.